Code Review Assignment_JrSDET .java

```diff
@@ -139,7 +139,7 @@ public HomePage(WebDriver driver) {
 
         public void verifyHomePage() {
             if (!driver.getCurrentUrl().equals("https://www.happyfox.com/home")) {
-                throw new IllegalStateException("Not on the home page");
+                throw new IllegalStateException("Not on the home page"); //code review

```

Comment:
This implementation is incorrect because it's doing exactly opposite of what it should be doing. correct way to implement is verifying whether currentURL matches with the HomePage url in the if block and if it matches that then there should be log which says that "Homepage is verified" and in else part exception can be handled or even better if we make use of try-catch.